feat(core/automerge): Allow a separate schedule for automerge (#15680)
renovatebot/renovate: docs/usage/configuration-options.md

}
```

## automergeSchedule

> The `automergeSchedule` option allows you to define times of week or month for automerging Renovate updates.

Reviewer: ```suggestion
Use the `automergeSchedule` option to define times of week or month during which Renovate may automerge its PRs.
```

> The default value for `automergeSchedule` is "at any time", which is functionally the same as declaring a `null` schedule.

Reviewer: ```suggestion
The default value for `automergeSchedule` is "at any time", which functions the same as setting a `null` schedule.
```

> i.e. Renovate will auto merge any merge-able update requests available.

Reviewer: ```suggestion
```

I think we can drop this, my suggestion

> Use the `automergeSchedule` option to define times of week or month during which Renovate may automerge its PRs.

covers the same ground.

> To configure this option refer to [`schedule`](https://docs.renovatebot.com/configuration-options/#schedule) as the syntax is the same.

Reviewer: ```suggestion
To configure this option refer to [`schedule`](#schedule) as the syntax is the same.
```

Would this work as a link within the file itself?

Author: yep, it works
